### ISSUE TITLE
Improve feedback for full storage

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/server#3471]
* set up a files external mount point that should be used by all users 
* set quota to 0 (so that they only can upload to the mount points)

It would now be nice to not show the warning `Your storage is full, files can not be updated or synced anymore!` if there are mount points specified (or show it less invasive).

<img width="571" alt="bildschirmfoto 2017-02-13 um 13 17 00" src="https://cloud.githubusercontent.com/assets/245432/22899072/26715a44-f1ef-11e6-8aa6-b82ccdb72cbd.png">

@nextcloud/designers Ideas? Maybe something like we use in the read only shares, which is not that invasive:

![bildschirmfoto 2017-02-13 um 13 22 24](https://cloud.githubusercontent.com/assets/245432/22899166/822dd538-f1ef-11e6-84a4-498337f50c38.png)

**Comment by skjnldsv** [comment]
If we implement [this](https://github.com/nextcloud/server/issues/3273#issuecomment-276599486), we could easily add a red warning or something as the icon :)

**Comment by MorrisJobke** [comment]
> If we implement this, we could easily add a red warning or something as the icon :)

This was also to make this less important, because there are also use cases where this is just unneeded info. So making this just a little grey text where usually the upload button is, would be the best I think.

**Comment by skjnldsv** [comment]
I feel like this is an important information though :/ 
But i'm okay with a text above the files :)

**Comment by juliushaertl** [comment]
Since we have the quota now in the left sidebar, we could just drop this message or move it to the quota section:

![image](https://user-images.githubusercontent.com/3404133/42163811-402d4072-7e04-11e8-8a4f-337bb68c2f45.png)

**Comment by skjnldsv** [comment]
I agree with @juliushaertl

**Comment by jancborchardt** [comment]
Yup! The showNotification should not be shown anymore. But we should also show this notice in the + (new file / folder menu) and deactivate those entries (except add folder cause that’s just reorganization, right?)

**Comment by szaimen** [comment]
I suppose this issue is still valid? If not, please close this issue!

**Comment by joshtrichards** [comment]
Related: #18552
